src/marketplace/sellers/semantic-version-check.md

```diff
@@ -0,0 +1,62 @@
+---
+group: marketplace-sellers
+title: Semantic Version Check
+---
+
+## Overview
+
+The Semantic Version Check (SVC) is a quality check that validates the change level of an extension submission against the change level declared by the extension developer.
+
+## Why we test?
+
+[Semantic Versioning](https://semver.org/) is a good way to communicate what kind of changes were introduced between software releases. This way, clients may estimate how risky it is to upgrade a particular piece of software and how much effort is required to adopt the new version.
+
+Magento follows Semantic Versioning and encourages all 3rd-party vendors to apply this versioning strategy for their Magento extensions as well.
+
+Magento Marketplace uses version information to fast-track the extension validation process for submissions that introduce patch level changes, as these are expected to be backwards-compatible bug fixes without any new, added functionality.
+
+## When we test?
+
+For this check to be applied, the following conditions must be met:
+
+1. A new submission must update an already existing version of an extension that has been previously published to the Marketplace storefront.
+1. The extension developer must declare the new submission is a "PATCH" level change.
+
+## What we are looking for?
+
+The SemVer check will analyze and determine the change level of the new submission and validate if, in fact, the change level is a "PATCH" level change.
+
+If the a submission is a "PATCH" level change, the submission is fast-tracked and is exempt from the Manual QA process. Otherwise, this check fails.
```

Comment:
```suggestion
If the submission is a "PATCH" level change, the submission is fast-tracked and is exempt from the Manual QA process. Otherwise, this check fails.
```